package.json

```diff
@@ -70,7 +70,6 @@
     "prettier": "2.6.1",
     "rimraf": "3.0.2",
     "ts-jest": "27.1.4",
-    "tslib": "2.3.1",
```

Comment:
This prevents duplication of ts helpers together with `importHelpers` in tsconfig.json

Reply:
brought back